lib/classification_lifecycle.rb

```diff
@@ -129,6 +130,10 @@ def add_project_live_state
     update_classification_metadata(:live_project, project.live)
   end
 
+  def add_lifecycled_at
+    classification.lifecycled_at = Time.now
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.